### ISSUE TITLE
RAdam in Tensorflow

### ISSUE THREAD

**Issue by phamnam95** [CyberZHG/keras-radam#18]
Are you going to do the RAdam in Tensorflow, which is similar to tf.train.AdamOptimizer? Thanks

**Comment by CyberZHG** [comment]
Added in `0.8.0`:

```python
from keras_radam.training import RAdamOptimizer

RAdamOptimizer(learning_rate=1e-3)
```

**Comment by phamnam95** [comment]
Thanks for your change. However, I still get errors when trying to use it with minimize method:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-797d05e4cdac> in <module>()
     61 #optimizer = tf.train.AdamOptimizer(1e-3)
     62 #optimizer = tf.contrib.estimator.clip_gradients_by_norm(optimizer,clip_norm=5.0)
---> 63 training_op = optimizer.minimize(loss)

~/ML_tensorflow/lib/python3.6/site-packages/tensorflow/python/training/optimizer.py in minimize(self, loss, global_step, var_list, gate_gradients, aggregation_method, colocate_gradients_with_ops, name, grad_loss)
    411 
    412     return self.apply_gradients(grads_and_vars, global_step=global_step,
--> 413                                 name=name)
    414 
    415   def compute_gradients(self, loss, var_list=None,

~/ML_tensorflow/lib/python3.6/site-packages/tensorflow/python/training/optimizer.py in apply_gradients(self, grads_and_vars, global_step, name)
    614           update_ops.append(processor.update_op(self, grad))
    615       if global_step is None:
--> 616         apply_updates = self._finish(update_ops, name)
    617       else:
    618         with ops.control_dependencies([self._finish(update_ops, "update")]):

~/ML_tensorflow/lib/python3.6/site-packages/keras_radam/training.py in _finish(self, update_ops, name_scope)
    254 
    255     def _finish(self, update_ops, name_scope):
--> 256         with ops.control_dependencies(update_ops):
    257             step, beta1_power, beta2_power = self._get_beta_accumulators()
    258             with ops.colocate_with(beta1_power):

~/ML_tensorflow/lib/python3.6/site-packages/tensorflow/python/framework/ops.py in control_dependencies(control_inputs)
   5424     return NullContextmanager()
   5425   else:
-> 5426     return get_default_graph().control_dependencies(control_inputs)
   5427 
   5428 

~/ML_tensorflow/lib/python3.6/site-packages/tensorflow/python/framework/ops.py in control_dependencies(self, control_inputs)
   4865           (hasattr(c, "_handle") and hasattr(c, "op"))):
   4866         c = c.op
-> 4867       c = self.as_graph_element(c)
   4868       if isinstance(c, Tensor):
   4869         c = c.op

~/ML_tensorflow/lib/python3.6/site-packages/tensorflow/python/framework/ops.py in as_graph_element(self, obj, allow_tensor, allow_operation)
   3794 
   3795     with self._lock:
-> 3796       return self._as_graph_element_locked(obj, allow_tensor, allow_operation)
   3797 
   3798   def _as_graph_element_locked(self, obj, allow_tensor, allow_operation):

~/ML_tensorflow/lib/python3.6/site-packages/tensorflow/python/framework/ops.py in _as_graph_element_locked(self, obj, allow_tensor, allow_operation)
   3883       # We give up!
   3884       raise TypeError("Can not convert a %s into a %s." %
-> 3885                       (type(obj).__name__, types_str))
   3886 
   3887   def get_operations(self):

TypeError: Can not convert a NoneType into a Tensor or Operation.

**Comment by CyberZHG** [comment]
I forgot a `return` in the optimizer... It should have been fixed in `0.9.0`.
